### PR TITLE
Update strings to approved copy

### PR DESCRIPTION
**Pull request by maxxcrawford**
This PR updates strings to approve copy from the [Figma spec](https://www.figma.com/file/5Om4NorNqTa2XE5ljBZilX/Mozilla-Privacy-Pack-Early-Access-MVP?node-id=26%3A5).

### CODE REVIEW

File: privaterelay/locales/en-US/app.ftl [hunk 1]
```diff
@@ -212,11 +213,27 @@ banner-choose-subdomain-label = Your domain is:
...
+
+# Variables:
+#   $number (string) - Maximum number of aliases a user can make on a free account. 
+error-premium-set-make-aliases = You must be a premium subscriber to make more than { $number } aliases
+error-premium-cannot-change-subdomain = You cannot change your subdomain
+error-premium-set-subdomain = You must be a premium subscriber to set a subdomain
+error-premium-set-create-subdomain = You must be a premium subscriber to create subdomain aliases
+error-subdomain-not-created = Subdomain could not be created, try something else
+error-subdomain-email-not-created = Email address with subdomain could not be created, try something else
+error-subdomain-select = You must select a subdomain before creating email address with subdomain
```
Comment (codemist) [inline privaterelay/locales/en-US/app.ftl:226]: Should it be `creating _an_ email address with subdomain?`

**Review by codemist — APPROVED** [review]
Looks good, I'm also not sure if my comment is the right way of saying it...

File: privaterelay/locales/en-US/app.ftl (hunk 1 above)
Comment (maxxcrawford) [reply privaterelay/locales/en-US/app.ftl:226]: Good catch!

Comment (maxxcrawford) [reply privaterelay/locales/en-US/app.ftl]: Revised to: 
> You must select a subdomain before creating a subdomain aliases

**Pull request merged**